grid/chat/members/profile.supr

```diff
@@ -0,0 +1,179 @@
+"""
+Chat Members
+Retrieve members of the workspace.
+"""
+name = "chat/members"
+version = "1.0.0"
+
+"""
+Get workspace members
+Retrieve paginated list of members
+"""
+usecase GetMembers {
+  input {
+    """
+    Workspace
+    Representation of a workspace. Can be ID or name, depending on the provider capability.
+    """
+    workspace string!
+    
+    """
+    Limit
+    Maximum number of records to retrieve
+    """
+    limit number
+    
+    """
+    Page
+    Identification of page with paginated results (cursor)
```

Comment:
```suggestion
    Page cursor retrieved from nextPage for results pagination
```

Reply:
This description is same across all chat use-cases 🤔

Reply:
I don't understand what's wrong with it 😄 what if user gets page cursor not from `nextPage`?

Reply:
My problem is that this description is not very readable and understandable in the first place. I am not arguing where you get the cursor, although if you get it elsewhere, you probably know what you are doing and don't need this description.